<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>498</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>813</width>
    <height>498</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>813</width>
    <height>498</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Sql-I-P_Auto</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo/window_icon.png</normaloff>logo/window_icon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Home</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="adminButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Admin Finder</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/admin.png</normaloff>:/icons/admin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Upload Script</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/upload.png</normaloff>:/icons/upload.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="themesButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Change theme</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/theme.png</normaloff>:/icons/theme.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>About us</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>641</width>
      <height>421</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>401</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Targetted scanning</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>60</y>
          <width>60</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Report:-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Check Vulnerability :-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>5</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Targetted scanning</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>551</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditx">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>59</y>
          <width>451</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>    Enter Url </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Multiple domain scanning</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>60</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Select Dork</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>index.php?=1</string>
         </property>
        </item>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>551</width>
          <height>201</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select Dork:-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Report:-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Multiple domain scanning</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="adminButton_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>70</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>470</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);</string>
      </property>
      <property name="placeholderText">
       <string>  Enter url eg.http://www. facebook.com</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Find Admin:-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Results:-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>501</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>YOU MUST WRITE PROTOCOLE HTTPS:// or HTTP:// ex: http://www.ex.com , ex :https://www.ex.com</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>370</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>YOU MUST WRITE / AT THE END OF THE SITE EX: www.example.com/</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>561</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 255);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AdminClearBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Admin Page Finder</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>281</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload Dorks</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>211</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>281</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Admin-Finder Script</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>211</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>0</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Uploads</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>591</width>
        <height>341</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="source">
       <url>
        <string>file:///C:/Users/Sandeep Sala/Documents/Sql-i-p-auto/x.txt</string>
       </url>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="copyrightlabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>@ Copyright 2019</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="themegroupBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>251</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Add New Theme ^-^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="theme1button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.426, y1:0.289773, x2:1, y2:0, stop:0.0681818 rgba(255, 85, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>DarkOrange</string>
     </property>
    </widget>
    <widget class="QPushButton" name="theme2button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.426, y1:0.289773, x2:1, y2:0, stop:0.0681818 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>LightBlue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="theme3button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.426, y1:0.289773, x2:1, y2:0, stop:0.0681818 rgba(0, 0, 127, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>DarkBlue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="theme4button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.426, y1:0.289773, x2:1, y2:0, stop:0.0681818 rgba(77, 77, 77, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>DarkGrey</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hideButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Host name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>546</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Ip Address :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
